Revamp of the Emergency Armory

ThatOneGuy42 opened the pull request:
http://i.imgur.com/lSjaBsh.png - What it looks like (minus the sidearm closet, that was deleted).

Meant to make the EA more Emergency and Less "Give everybody weapons." Tactical Armory was made far lighter, armor replaced with SCG Gear, pulse carbines removed, far less guns, and a lot more emergency gear.

ThatOneGuy42 committed:
Redo of the Emergency Armory

Revamps the Emergency Armory. Less weapons, less armor, more emergency.

redstryker commented:
Makes everything more general emergency-oriented, not necessarily shooty-emergency oriented... I like it!

xales commented:
Looks good, why is there a merge commit though?

ThatOneGuy42 commented:
To be honest I have no idea.

AtomicWorm commented:
"Revamp". At least call it what it is. A nerf.

IDTia commented:
Some things though that would also help improve it, removing the photocopier and moving the stuff on the front desk to the side desk. Makes equipment sort of a pain to pass out sometimes and the photocopier gets in the way of you carry a crate full of equipment behind you. 

Two syringe guns though to be honest nobody uses them, or moreso rarely if ever. One thing that would make the armory attractive and more relevant to medical is to add one or two combat aid kits, because medicine, but that's just a personal opinion. 

Happy to see all the tac stuff gone though. Hurray.

Datraen commented:
https://www.merriam-webster.com/dictionary/revamp
Revamp is correct either way.

I like the focus on emergency situations, but it's important to keep in mind it's a military ship that should be stocked appropriately. 

Size of the diff makes me wonder if mapmerge was used.

Edited for better formatting.

N8-Toe commented:
Two medical crates may be overkill. maybe replace one with a crate of metal, glass, and reinforced glass. also all of that empty space should have something to fill it. maybe racks with radiation supplies, one with spare things like cuffs and tasers, and maybe a rack of bio suits to help fill the empty space. long shot but I think stun rifles should stay.

AtomicWorm commented:
So what balance changes do mercs get to account for the loss of like 70% of the Torch's firepower?

serenemelody commented:
Oh boo hoo. Now the Torch can't dogpile every baddie with lasers from one central location. Truly this is the worst timeline.

xales commented:
Silly GitHub

ThatOneGuy42 commented:
There's still what's in the Security Armory, and there are in fact 12 guns there in the Tactical Area. As well, each head comes equipped with an egun in their locker. That means around 20 or so people on the Torch can be equipped with a form of weaponry in case of an outside threat such as Boarders (the only time any such thing might be necessary cause GQ isn't a thing).

Datraen reviewed (approved):
Diff is no longer huge, it's been too long since I've hit this button.

Kasuobes commented:
Inconsistent decals from the screenshot, some racks have checkered outlines underneath while others don't. 

Misaligned scrubber/vent pair in E-armory.

I question the need for two mech chargers, considering they never get used in the first place because getting a mech in there is a fuckfest to begin with.

Missing fire alarm and cameras in E-armory.

Someone check the lighting to see if the strip between the suit units and the T-armory door is too dark. Entrance to E-armory is also brighter than the sun with 3 light tubes.

No laser carbines (or any beam weapon with penetration) means anyone wearing ablative armor (full suit available in sec) is effectively invulnerable unless flashed or gassed or physically hit.

Two syringe guns = incoming chloral dart meta.

Chiquiii commented:
i like the change, but it feels like the tactical armory is basically wasting space on armor storage.

Crushtoe commented:
I intend to do further edits after @ThatOneGuy42. The biggest thing is making sure it gets nerfed from pulse-r-us.

ThatOneGuy42 committed:
fixes some things

Kasuobes commented:
+2,505 −2,523

You what?

ThatOneGuy42 commented:
Well I was gonna do mapmerge first, but hadn't gotten the chance to yet. Changes including adding fire alarms, camera, switching up lighting, and adding the grey tiles for consistency. Also replaced pulse pistols and ion pistols with laser carbines and replaced taser rifles with stun carbines per suggestion.
